### PR TITLE
remote transactor

### PR DESCRIPTION
**Pull request by grischoun**
- Fixes conflicts of the old PR https://github.com/replikativ/datahike/pull/281.
- Changes pom version in preparation of a new deployment.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <groupId>io.replikativ</groupId>
   <artifactId>datahike</artifactId>
   <packaging>jar</packaging>
-  <version>0.3.6-SNAPSHOT</version>
```
Comment (kordano) [inline pom.xml:7]: why was the snapshot removed?

Comment (grischoun) [reply pom.xml:7]: Right. I’ll restore it.

Comment (grischoun) [reply pom.xml]: Fixed now.

**Review by kordano — APPROVED** [review]
looks good, thanks

**Pull request merged**